### PR TITLE
Deploy script patch

### PR DESCRIPTION
**Pull request by imOnVacation**
Trying to update deploy script patch to development branch

### CODE REVIEW

**Comment by imOnVacation** [comment]
@NAlexH2  
I'm trying to merge the script patch into development. The only thing i would change before merge into development is to keep "sentiment" package, and "sentiment" and "bad-words" on devDependencies.

**Comment by imOnVacation** [comment]
**PLEASE DO NOT merge this PR yet!** Not until script-patch successfully merge to main and MVP

**Comment by NAlexH2** [comment]
> @NAlexH2 I'm trying to merge the script patch into development. The only thing i would change before merge into development is to keep "sentiment" package, and "sentiment" and "bad-words" on devDependencies.

@imOnVacation  Hmm well lets merge it into MVP first then do this PR and I'll add them back in then AFTER the first PR is approved. They do not need to be apart of MVP.

**Review by NAlexH2 — APPROVED** [review]

**Pull request merged**
